docs/dev/install.rst

```diff
@@ -129,7 +129,7 @@ running:
 
 .. code-block:: bash
 
-   python manage.py runserver
+   ./manage.py runserver localhost:8000
```

Comment:
Our current FXA setup rely on the fact that developers run their instances at http://localhost:8000 (this is also a default value for the django runserver command).